apps/modernization-api/src/test/resources/features/patient/PatientProfile.feature

```diff
@@ -0,0 +1,18 @@
+@patient-profile
+Feature: Patient Profile
+
+  Background:
+    Given I have a patient
+
+  Scenario: I can retrieve a profile by patient identifier
+    Given I have the authorities: "FIND-PATIENT" for the jurisdiction: "ALL" and program area: "STD"
+    When a profile is requested by patient identifier
+    Then the profile is found
+
+  Scenario: I can retrieve a profile by short identifier
+    Given I have the authorities: "FIND-PATIENT" for the jurisdiction: "ALL" and program area: "STD"
+    When a profile is requested by short identifier
+    Then the profile is found
+
+  Scenario: I cannot retrieve a profile without proper authorities
+    Then the profile is not found
```

Comment:
still needs the `When a profile is request by patient identifier` right? Could add a second to do `When a profile is requested by short identifier` as well.